src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -1209,9 +1220,37 @@ namespace AirflowNetworkBalanceManager {
 			// Wind coefficient == Surface-Average does not need inputs of external nodes
 			CurrentModuleObject = "AirflowNetwork:MultiZone:ExternalNode";
 			AirflowNetworkNumOfExtNode = GetNumObjectsFound( CurrentModuleObject );
+			if ( AnyLocalEnvironmentsInModel ) {
+ 				CurrentModuleObject = "OutdoorAir:Node";
+ 				AirflowNetworkNumOfOutAirNode = GetNumObjectsFound( CurrentModuleObject );
+ 				AirflowNetworkNumOfExtNode += AirflowNetworkNumOfOutAirNode;
+ 			}
 			if ( AirflowNetworkNumOfExtNode > 0 ) {
 				MultizoneExternalNodeData.allocate( AirflowNetworkNumOfExtNode );
-				for ( i = 1; i <= AirflowNetworkNumOfExtNode; ++i ) {
+				if ( AnyLocalEnvironmentsInModel ){
+ 					CurrentModuleObject = "OutdoorAir:Node";
+ 					for ( i = AirflowNetworkNumOfExtNode - AirflowNetworkNumOfOutAirNode + 1; i <= AirflowNetworkNumOfExtNode; ++i ) {
+ 						GetObjectItem( CurrentModuleObject, i - ( AirflowNetworkNumOfExtNode - AirflowNetworkNumOfOutAirNode ), Alphas, NumAlphas, Numbers, NumNumbers, IOStatus, lNumericBlanks, lAlphaBlanks, cAlphaFields, cNumericFields );
```

Comment:
@jasondegraw @lgu1234 I put it here as the original allocation of the external nodes was here, and I was hoping to allocate more (1) to the same structure, (2) under the same situation - AirflowNetworkSimu.iWPCCntr == iWPCCntr_Input. 

@lgu1234 I switched the order.